Cairo Desktop/CairoDesktop.Configuration/app.config

```diff
@@ -145,6 +145,15 @@
       <setting name="EnableMenuBar" serializeAs="String">
         <value>True</value>
       </setting>
+      <setting name="HideTaskbarLabels" serializeAs="String">
```

Comment:
Could you inverse the setting to be "Show labels" rather than hide, and default that to true? This makes the setting consistent with other taskbar settings.

Reply:
Sure, makes sense.